osbs/conf.py

```diff
@@ -356,6 +356,9 @@ def get_git_push_username(self):
     def get_build_image(self):
         return self._get_value("build_image", self.conf_section, "build_image")
 
+    def get_build_imagestream(self):
+        return self._get_value("build_imagestream", self.conf_section, "build_imagestream")
```

Comment:
We also need `--build-imagestream` cmdline argument in `cli/main.py`. Perhaps make it [mutually exclusive](https://docs.python.org/2/library/argparse.html?highlight=argparse#mutual-exclusion) with `--build-image`?